### ISSUE TITLE
ViewSonic LS700HD not powering on

### ISSUE THREAD

**Issue by tyler351** [bitfocus/companion-module-pjlink#49]
This module was working for a while but as of 2.3.1+4641-dc01ac7c, it stopped working when I attempt to power on this projector. I can send raw commands to the projector and it works as intended so I know the projector is responding to pjlink stuff as of right now. At this point, I'm trying to determine what may be the cause of it not working when trying it with companion. 

We have a few other pjlink devices (Panasonic and an other Viewsonic projector that works just fine.) We are running companion on a linux VM and just using the web buttons to trigger the commands.

Any help would be appreciated.

**Comment by istnv** [comment]
There are several recent updates to the module (2.4, 3.0). Re-open if this is still a problem

**Comment by tyler351** [comment]
Definitely having some issues with 2.4.2. Both of the projectors we have controlled via pjlink have stopped working completely with v2.4.2. I installed beta-v3 just to see if that had the same issues, but beta-v3 actually works. 
The projectors we are working with are a viewsoic LS800HD and LS700HD. Both support commands via PJLINK and operate properly when using the PJLINK testing application found on the JBMIA website.
